scss/_base_forms-tick-elements.scss

```diff
@@ -18,40 +23,114 @@ $box-offsets-top: (
   table-header: 1.9,
 ) !default;
 
-@mixin vf-b-tick-elements {
-  // Default form checkbox and radio styles
-  %vf-tick-elements {
+@mixin vf-b-tick-elements-definitions {
+  %vf-hidden-tick-input {
+    bottom: 0;
+    float: none;
+    height: $form-tick-box-size;
+    margin: 0;
     opacity: 0;
     position: absolute;
+    width: $form-tick-box-size;
+  }
 
-    label & {
-      // some basic styling for checkboxes inside labels
-      float: left;
-      margin: $spv-inner--x-small $sph-inner 0 0;
+  %vf-pseudo-tick-box {
+    padding-left: $sph-inner + $form-tick-box-size;
+    position: relative;
+
+    &::before,
+    &::after {
+      $properties: #{background-color, border-color};
+      @include vf-animation($properties);
+
+      position: absolute;
+    }
+
+    // container
+    &::before {
```

Comment:
An idea - feel free to ignore as this may complicate things. But should we hardcode the styling of container / tick onto before/after? Or is it likely that we may want to change that because of things like the *required functionality? 

One thing I was thinking of was creating a mixin that allows you to attach to before/after using arguments with default values like` before: container, after: tick ` maybe?

Reply:
I would prefer not to complicate things until it's needed. I don't expect `required` to be a problem, because ::before and ::after of the tick will be on different element (`<span class="p-checkbox__label">`) than ::after from the required (on `<label class="p-checkbox">`.

We can always restructure the placeholders/mixins if needed, but hopefully what is here will allow us to build components, and then in 3.0, if we drop base styling, we won't need that much of reuse of checkboxes.